service/matching/tasklist/task_list_limiter.go

```diff
@@ -22,6 +22,7 @@ type taskListLimiter struct {
 	scope             metrics.Scope
 	ttl               time.Duration
 	minBurst          int
+	overrideRPS       func() float64
```

Comment:
Since we'll be using it only within task_list_manager.go we no longer need to add this here. I think this file ultimately will be unchanged, apologies for the iteration. I think David's proposal (applying it to incoming values) for how to structure it is better than mine.